src/pcre2_compile_class.c

```diff
@@ -1752,6 +1774,8 @@ whole class was negated and whether there were negative specials such as \S
 (non-UCP) in the class. Then copy the 32-byte map into the code vector,
 negating it if necessary. */
 
+PCRE2_ASSERT(classbits == cb->classbits.classbits);
```

Comment:
We could lose this assertion if you want. We only ever assign once to the classbits variable (it's const) so there's really not much to check.

Reply:
Ok. I have added a comment about it as well.